language/unit_tests/test_client.py

```diff
@@ -26,19 +26,25 @@ def _make_one(self, *args, **kw):
         return self._get_target_class()(*args, **kw)
 
     def test_ctor(self):
+        import mock
+        from oauth2client.client import GoogleCredentials
         from google.cloud.language.connection import Connection
 
-        creds = _Credentials()
+        creds = mock.Mock(spec=GoogleCredentials)
+        creds.create_scoped_required.return_value = False
```

Comment:
@tseaver Before I write the helper I want to confirm how you mean with `kwargs`. ISTM it must be a layer of nesting:

```python
creds = mock.Mock(
    spec=GoogleCredentials,
    create_scoped_required=mock.Mock(return_value=False))
```

unless there is some other syntax I'm unaware of?